app/models/court_application.rb

```diff
@@ -3,39 +3,33 @@
 class CourtApplication < ApplicationRecord
   include BuilderMappable
   include CourtCentreRelatable
-  belongs_to :application_type, class_name: "CourtApplicationType"
+
   belongs_to :court_application_party
-  belongs_to :court_application_outcome, optional: true
   belongs_to :court_application_payment, optional: true
-  has_many :court_application_respondents
+  belongs_to :court_application_type
+
   has_many :judicial_results
 
```

Comment:
`has_many :court_application_party` ?

Reply:
`belongs_to` is ok here. More than one field refers to the CourtApplicationParty class, however in each case the field refers to only one instance of that class

Reply:
So `respondents` is an array of courtApplicationPartys -I think this was the one I was thinking of. Not sure how you'd represent some fields having only one and another having an array of them though...